src/web/app/src/config.ts

```diff
@@ -24,31 +21,26 @@ const title = `Telescope`;
 const description = `A tool for tracking blogs in orbit around Seneca's open source involvement`;
 const author = `SDDS Students and Professors`;
 const keywords = `Telescope, Seneca College, Open Source, Programming, Software Development, Blogs, Education, CDOT, SDDS, Students, Professors, Learning, Teaching`;
-const image = `${telescopeUrl}/logoImages/apple-splash-1334-750.jpg`;
+const image = `/logo.svg`;
```

Comment:
Is an SVG compatible with a JPG in the ways we use this?

Reply:
So the `image` is actually being used here
https://github.com/Seneca-CDOT/telescope/blob/6046273fba4b89ce6281f7a34cafbdec7ef222dc/src/web/app/src/pages/_document.tsx#L6-L36
We tested it out by starting the services and there wasn't a problem